rebar.config

```diff
@@ -1,3 +1,8 @@
+{deps, [
+    {goldrush, ".*",
+        {git, "https://git-wip-us.apache.org/repos/asf/couchdb-goldrush.git",
+        {branch, "master"}}}
```

Comment:
Will be conflict with [root repository](https://github.com/apache/couchdb/blob/master/rebar.config.script#L39) - either need to change "master" to "0.1.6" tag or remove goldush from apache/couchdb rebar.config.script.